src/main.ts

```diff
@@ -62,10 +62,10 @@ core.debug(JSON.stringify({executeUrl, context}, null, 2))
 try {
   const response = await fetch(executeUrl, {
     headers: {
-      'Content-Type': 'application/json'
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${token}`
```

Comment:
I would probably NOT send it in the oauth authorization header specifically, but something like X-API-Key or something, that way it's less confusing I think. Cf. also for example the swagger docs that recommend something like that: https://swagger.io/docs/specification/authentication/api-keys/.


Or if we want to stay with the Authorization header, maybe use `Org` or something as the prefix. Basically something to differentiate it from a "normal" oauth token.

Reply:
Ok I found some other page that recommended to do it like this but I can change it also